src/meta/src/hummock/compaction/level_selector.rs

```diff
@@ -82,6 +88,81 @@ impl DynamicLevelSelector {
             inner: LevelSelectorCore::new(config, overlap_strategy),
         }
     }
+
+    fn calculate_l0_overlap(
```

Comment:
Please corret me if I am wrong, this method calculates "total size of non-pending L0 file size + total size of base level SSTs overlapping with the non-pending L0 SSTs". Based on this, I have the following questions:
1. Shouldn't we use `if !handlers[0].is_pending_compact(&table_info.id)` in L102 so that we are actually use non-pending L0 SSTs to calculate overlap?
2. `output_files` in L108 represents the pending SSTs in base level that output to base level itself. IIUC, we should consider these SSTs in next_level_size instead of ignoring them in L116. To be more precise, I think we should consider non-pending SSTs and pending SSTs with target level == base level for next_level_size.
3. Do we intentionally ignore L0 sub-level compaction?